src/kOS.Safe/Encapsulation/PIDLoop.cs

```diff
@@ -219,7 +225,8 @@ public ScalarValue Update(ScalarValue sampleTime, ScalarValue input)
                             unWinding = false;
                         }
                     }
-                    iTerm = ITerm + error * dt * Ki;
+                    iTerm = ITerm + lastIError * dt;
+                    lastIError = error * Ki;
```

Comment:
Can you explain why is it supposed to be better to time-shift the I term one iteration like this so it's using the previous error value instead of the current one from this iteration?  The issue #2919 only has one sentence in the description which doesn't explain why it's desired to delay it like this.

Reply:
I have added some explanations to PR comments, but I shouldn't have changed the integration method, so I'll revert this.